webview.h

```diff
@@ -1138,7 +1141,454 @@ class win32_edge_engine {
 using browser_engine = win32_edge_engine;
 } // namespace webview
 
-#endif /* WEBVIEW_GTK, WEBVIEW_COCOA, WEBVIEW_EDGE */
```

Comment:
Isn't this redundant? Adding an extra `#endif ... #elif` shouldn't be necessary here since the rest of the file already separates it by `#elif`s.

Reply:
Either you edited it or I misread the code. Either way, it is fine now.

Reply:
It was always like the other ones from the start... The only thing should be taken with care, if I remember well, is that Android define must go before Linux, otherwise it tries to build Gtk backend instead.